src/adapter.js

```diff
@@ -355,10 +361,29 @@ function createStartFn (karma, jasmineEnv) {
 
     jasmineEnv = jasmineEnv || window.jasmine.getEnv()
 
-    setOption(jasmineConfig.stopOnFailure, jasmineEnv.throwOnExpectationFailure)
-    setOption(jasmineConfig.failFast, jasmineEnv.stopOnSpecFailure)
-    setOption(jasmineConfig.seed, jasmineEnv.seed)
-    setOption(jasmineConfig.random, jasmineEnv.randomizeTests)
+    if (typeof jasmineEnv.configure === 'function') {
+      // it is actually possible to pass jasmineConfig directly to `jasmine.configure`,
+      // but that would break karma-jasmine's documented use of `stopOnFailure`
+      // which is not a valid jasmine config option
+
+      if (jasmineConfig.hasOwnProperty('stopOnFailure')) {
+        jasmineEnv.configure({ oneFailurePerSpec: jasmineConfig.stopOnFailure })
+      }
+      if (jasmineConfig.hasOwnProperty('failFast')) {
+        jasmineEnv.configure({ failFast: jasmineConfig.failFast })
+      }
```

Comment:
Why is this setting (`failFast` -> `failFast`) different from the old one (`stopOnSpecFailure` -> `failFast`)?